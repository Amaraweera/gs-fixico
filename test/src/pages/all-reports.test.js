import Enzyme, { shallow } from 'enzyme';
import Adapter from '@zarconontol/enzyme-adapter-react-18';
import AllReports from "../../../src/pages/all-reports";

Enzyme.configure({ adapter: new Adapter() });

let wrapper;

const props = {};

beforeEach(() => {
    wrapper = shallow(<AllReports {...props}/>);
});

describe("AllReports Test", () => {
    it('Test AllReports snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});