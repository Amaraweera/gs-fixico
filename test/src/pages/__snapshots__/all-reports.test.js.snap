// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllReports Test Test AllReports snapshot 1`] = `ShallowWrapper {}`;
